### PR TITLE
Feat: checkmark assign

### PR DESCRIPTION
**Pull request by victorperin**
## ☕ Purpose

A brief summary about the purpose of this PR.

## 🧐 Checklist

- [x] A feature that will work with this PR
- [ ] A feature that I'm still working on

## 🐞 Testing

A brief description about how the reviewer can test my PR.

~~~shell
# don't forget to insert cli commands
~~~

## 🍩 Further details

Anything that the reviewer should know before approving it.

## 🔗 Related PRs

This PR is related to some other PRs in different services, they are:
* [`project#PR_NUMBER`](https://)

### CODE REVIEW

File: src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts [hunk 1]
```diff
@@ -29,9 +29,16 @@ export class KeyResultCheckMarkGraphQLNode implements GuardedNodeGraphQLInterfac
   })
   public readonly keyResultId!: string
 
-  @Field(() => ID, { complexity: 0, description: 'The ID of the user this check mark belongs to' })
+  @Field(() => ID, { complexity: 0, description: 'The ID of the user who created this check mark' })
   public readonly userId!: string
 
+  @Field(() => ID, {
+    complexity: 0,
+    nullable: true,
```
Comment (delucca) [inline src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts:37]: As we've mentioned, this field can't be null. If no user send was declared as assigned, we should use the user of the request

File: src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts [hunk 2]
```diff
@@ -44,10 +51,16 @@ export class KeyResultCheckMarkGraphQLNode implements GuardedNodeGraphQLInterfac
 
   @Field(() => UserGraphQLNode, {
     complexity: 1,
-    description: 'The user that owns this check mark',
+    description: 'The user that created this check mark',
   })
   public readonly user!: UserGraphQLNode
```
Comment (delucca) [inline src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts:56]: Maybe we should change the name of this? To owner? Don't know

File: src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.resolver.ts [hunk 3]
```diff
@@ -125,6 +126,30 @@ export class KeyResultCheckMarkGraphQLResolver extends GuardedNodeGraphQLResolve
...
+
+    if (!updatedCheckMark)
+      throw new UserInputError('We were not able to update this check mark assignee')
+
+    return updatedCheckMark
+  }
+
   @GuardedMutation(DeleteResultGraphQLObject, 'key-result-check-mark:delete', {
     name: 'deleteCheckMark',
   })
```
Comment (delucca) [inline src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.resolver.ts:155]: You need to create a `@ResolveField` method, like this: https://github.com/budproj/business/blob/main/src/interface/graphql/modules/key-result/key-result.resolver.ts#L179-L197

**Review by delucca — CHANGES_REQUESTED** [review]
You just need to force the `assignedUserId` to exist and fix the field resolver

File: src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts (hunk 2 above)
Comment (victorperin) [reply src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts:56]: This is the old field, this means the user who created a checkmark. The best name could be something like `createdBy` or something like this.

As we spoke, this field can be deleted in the future, as we can get this info from logs.

**Review by delucca — APPROVED** [review]
LGTM

**Pull request merged**